vulture_allowlist.py

```diff
@@ -205,4 +205,6 @@
 _.environment  # unused attribute (monkey/monkey_island/cc/services/telemetry/processing/system_info_collectors/environment.py:10)
 _.instance_name  # unused attribute (monkey/common/cloud/azure/azure_instance.py:35)
 _.instance_name  # unused attribute (monkey/common/cloud/azure/azure_instance.py:64)
+# TODO: Remove this when adding LINUX specific bb test
+LINUX  # unused variable (envs/monkey_zoo/blackbox/test_blackbox.py:56)
```

Comment:
If this is unused, do we even need to add it at all? Rather than hoping we remember to remove this (if we ever add a linux-specific test), let's just not add the LINUX constant.